Frameworks/Social.Xaml/ShareSheet.xaml

```diff
@@ -0,0 +1,20 @@
+<ContentDialog
+    x:Class="Social.Xaml.ShareSheet"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Social.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Facebook"
+    PrimaryButtonText="Post"
+    SecondaryButtonText="Cancel"
```

Comment:
sure; prob ok until we need to localize

---
In reply to: [107997791](https://github.com/Microsoft/WinObjC/pull/2314#discussion_r107997791) [](ancestors = 107997791,107995907,107995837)